terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl

```diff
@@ -184,10 +184,33 @@ else
     fi
     if upgrade_automate; then
         if [[ "${airgapped}" == "true" ]]; then
-            chef-automate upgrade run --airgap-bundle /tmp/automate.aib
+            # chef-automate upgrade run --airgap-bundle /tmp/automate.aib
+            ERROR=$(chef-automate upgrade run --airgap-bundle /tmp/automate.aib 2>&1 >/dev/null) || true
+            if echo "${ERROR}" | grep 'This is a Major upgrade'; then
+            echo "y
+    y
+    y
+    y
+    y" | chef-automate upgrade run --major --airgap-bundle /tmp/automate.aib
+
+                # NOTE: This is a hack
+                # The hack above was no longer good enough because we have a thing that needs
+                # to be updated that isn't a service
+                sleep 45
+
+                #shellcheck disable=SC2154
+                wait_for_upgrade
+                chef-automate post-major-upgrade migrate --data=PG -y
```

Comment:
This works for the current upgrade, how about future upgrades? Does this command gracefully allow the process to continue? If this command fails, the further code won't execute